collections/made-in-india/index.md

```diff
@@ -13,6 +15,7 @@ items:
  - shivammathur/setup-php
  - spaceuptech/space-cloud
  - uvdesk/community-skeleton
+ - kadalu/moana
```

Comment:
@amarts Please remove this project, since it's a part of Kadalu itself and wouldn't add much value to the overall collection. Many open-source projects usually have multiple repos and including only the main one would be sufficient.

Reply:
done